mypy.ini

```diff
@@ -0,0 +1,17 @@
+[mypy]
```

Comment:
Could we add it to pyproject.toml instead? Just to avoid having a yet another config file.

Reply:
@efiop, it does not support `pyproject.toml` yet, but we can add it to `setup.cfg`.